src/main.ts

```diff
@@ -24,6 +24,6 @@ async function bootstrap() {
     defaultVersion: ['1'],
   });
 
-  await app.listen(3000);
+  await app.listen(3001);
```

Comment:
- use [config/configuration.ts](https://docs.nestjs.com/techniques/configuration#custom-configuration-files) to setup listen port
- declare DEFAULT_PORT = 3001